.github/workflows/nuget-ci-cd.yml

```diff
@@ -75,7 +75,7 @@ jobs:
         if-no-files-found: error
 
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
why can't we use ubuntu 24? At if we don't fix it then we should document why we're not using latest so someone knows what to look out for when upgrading.

Reply:
Just pushed a fix for this. Now the build uses `dotnet pack` instead of nuget.exe, which simplifies cross-platform building quite a bit and allows us to run on `ubuntu-latest` again. Also, the use of `dotnet pack` allows us to use the `buildTransitive` feature (new in NuGet 5.0) which will allow packages consuming Chorus to be able to specify the Mercurail4ChorusDestDir variable without having to consume the SIL.Chorus.Mercurial package directly. This, in turn, will remove the potential for version conflicts on the SIL.Chorus.Mercurial package (which I've run into before).

So thanks for this comment; it pushed me to fix a long-standing problem.